### ISSUE TITLE
connectDragPreview not work when it is directly inside of connectDropTarget

### ISSUE THREAD

**Issue by mmillet** [react-dnd/react-dnd#1465]
**Describe the bug**
If connectDragPreview is directly inside of connectDropTarget, it works bad. But the old version 7.7.0 works well.
`connectDropTarget(connectDragPreview(/*...*/))`

Otherwise if I wrap a DOM node(eg. `<span>`) with connectDragPreview, it works well again:
`connectDropTarget(<span>{connectDragPreview(/*...*/)}</span>)`

Maybe it's my fault, look forward your reviewing.

**Reproduction**
[Live  Reproduction](https://codesandbox.io/embed/connectdragpreview-connectdroptarget-infng)

1. Click on 'Drag me by the handle GOOD', it works well
2. Click on 'Drag me by the handle BAD', see error

**Expected behavior**
Click on 'Drag me by the handle BAD' works well

**Screenshots**
![Screenshots](https://thumbimg.dealmoon.com/dealmoon/52d/1a2/b21/0703df49d401eb409e63dd9.png)

**Desktop:**
- OS:  macOS 10.14.5
- Browser: chrome
- Version:
    react: 16.8.4
    react-dom: 16.8.4
    react-dnd: 9.3.1
    react-dnd-html5-backend: 9.3.1

**Comment by darthtrevino** [comment]
I’ll see if I can tackle this next week. As a workaround - chaining works if you use refs:

```jsx
let ref = useRef()
connectDragSource(connectDragPreview(ref))
return <div ref={ref} />
```

**Comment by mmillet** [comment]
@darthtrevino  The workaround works well, and waiting for your tackling. Thank you!

**Comment by PKaPI** [comment]
The problem is not solved in version 9.3.2.connectDragPreview not work when it is directly inside of connectDropTarget
